app/models/efile_submission.rb

```diff
@@ -128,9 +128,15 @@ def generate_irs_address
 
   def generate_form_1040_pdf
```

Comment:
since this is now the entire tax return (includes more than just the 1040, maybe we can update this to "generate_tax_return_pdf" or "generate_filing_pdf"